### PR TITLE
fix: ensure Gradle 6 support, update test matrix

### PR DESCRIPTION
**Pull request by kyegupov**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Updates the test matrix to ensure Gradle 6 support.
Also:
* introduces testing for Java 13 and Node 12
* drops non-LTS (unsupported) Java versions
* minimizes the usage of Gradle Wrapper in tests
* suppresses specific tests for unsupported JDK/Gragle versions

Drive-by: clarify the code used to extract the Gradle version.

### CODE REVIEW

File: test/fixtures/gradle-kts/gradlew [hunk 1]
```diff
@@ -1,172 +0,0 @@
-#!/usr/bin/env sh
```
Comment (sfat) [inline test/fixtures/gradle-kts/gradlew]: is there a reason why this is not needed anymore?

Comment (kyegupov) [reply test/fixtures/gradle-kts/gradlew]: yeah, let's try to use system Gradle from the test matrix

File: .travis.yml [hunk 2]
```diff
@@ -17,49 +17,58 @@ before_script:
...
+  - stage: Different Node/Java versions + most recent Gradle version
+    # We support Node 6+, Java 8+, Gradle 2.14+
+    # Tested Java versions are usually: 8 (LTS), 11 (LTS), 13 (latest)
+    # (see https://www.oracle.com/technetwork/java/java-se-support-roadmap.html)
+    # LTS version are also the most popular ones: https://www.baeldung.com/java-in-2019
+    # We use sdkman to install Java. 
+    # AdoptOpenJDK distibution with HotSpot VM is chosen as the most "classic" option.
     node_js: 6
-    env: GRADLE_VERSION=5.3.1 JDK=9
+    env: GRADLE_VERSION=6.0 JDK=8 Node.js=6
```
Comment (lili2311) [inline .travis.yml]: Please keep 5.3 as well though, good to have something from 5

Comment (kyegupov) [reply .travis.yml]: 5.3 was moved to the "older versions". Do you think it should be kept here?

Comment (lili2311) [reply .travis.yml]: as long as it is being tested older version is fine :)

File: README.md [hunk 3]
```diff
@@ -15,35 +15,37 @@ This plugin provides dependency metadata for Gradle projects that use `gradle` a
 
 # Support
 
-❓No issues expected but test not updated yet to verify
-✅Supported and verified with tests
+❌ Not supported
+❓ No issues expected but hit regularly tested
```
Comment (orsagie) [inline README.md]: s/hit/not?

Comment (lili2311) [inline README.md]: `No issues expected but hit regularly tested` not sure what this means?

Comment (kyegupov) [reply README.md]: fixed

Comment (kyegupov) [reply README.md]: typo

**Pull request merged**